unit-tests/live/streaming/test-y16.cpp

```diff
@@ -0,0 +1,52 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+//#test:device D400*
+
+#include "../../catch.h"
+#include "../../unit-tests-common.h"
+
+using namespace rs2;
+
+rs2::stream_profile get_ir_y16_profile(rs2::depth_sensor depth_sensor)
+{
+    auto depth_profiles = depth_sensor.get_stream_profiles();
+    rs2::stream_profile ir_profile;
+    for (auto& p : depth_profiles)
+    {
+        if (p.format() == RS2_FORMAT_Y16)
+        {
+            ir_profile = p;
+            break;
+        }
+    }
+    return ir_profile;
+}
+
+// Y16 CONFIGURATION TESTS
+TEST_CASE("Y16 streaming", "[live]") 
+{
+    rs2::context ctx;
+    auto list = ctx.query_devices();
+    REQUIRE(list.size());
+    auto dev = list.front();
```

Comment:
This section can be replaced with
``` find_first_device_or_exit() ```
See https://github.com/IntelRealSense/librealsense/blob/1b9a40da35f20a46b2113cabd920ce37f757743b/unit-tests/live/live-common.h#L51

Reply:
Done